docs/core/setup/linux-package-manager-rhel7.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Install .NET Core on Linux RHEL 7 package manager
+description: Use a package manager to install .NET Core SDK and runtime on RHEL 7.
+author: thraka
+ms.author: adegeo
+ms.date: 11/06/2019
+---
+
+# RHEL 7 Package Manager - Install .NET Core
+
+> [!div class="op_single_selector"]
+> - [RHEL 7 - x64](linux-package-manager-rhel7)
+> - [Ubuntu 16.04 - x64](linux-package-manager-ubuntu-1604)
+> - [Ubuntu 18.04 - x64](linux-package-manager-ubuntu-1804)
+> - [Ubuntu 19.04 - x64](linux-package-manager-ubuntu-1904)
+> - [Debian 9 - x64](linux-package-manager-debian9)
+> - [Debian 10 - x64](linux-package-manager-debian10)
+> - [Debian 10 - x64](linux-package-manager-debian10)
+> - [Fedora 29 - x64](linux-package-manager-fedora29)
+> - [Fedora 30 - x64](linux-package-manager-fedora30)
+> - [CentOS 7 - x64](linux-package-manager-centos7)
+> - [OpenSUSE 42.3 - x64](linux-package-manager-opensuse423)
+> - [OpenSUSE 15 - x64](linux-package-manager-opensuse15)
+> - [SLES 12 - x64](linux-package-manager-sles12)
+> - [SLES 15 - x64](linux-package-manager-sles15)
+
+This article describes how to use a package manager to install .NET Core on RHEL 7.
+
+## Register your Red Hat subscription
```

Comment:
can't this be embedded in the other articles? Use an include file if it can be used my more than one article. I don't think it deserves its own.

Reply:
We're experimenting with how to present this. This is replacing these pages: https://dotnet.microsoft.com/download/linux-package-manager/ubuntu16-04/sdk-current